tabs/configuration.js

```diff
@@ -436,6 +451,104 @@ TABS.configuration.initialize = function (callback, scrollPosition) {
         $('input[name="currentoffset"]').val(BF_CONFIG.currentoffset);
         $('input[name="multiwiicurrentoutput"]').prop('checked', MISC.multiwiicurrentoutput);
 
+        var escProtocols = {
+            0: {
+                name: "STANDARD",
+                defaultRate: 400,
+                rates: {
+                    50: "50Hz",
+                    400: "400Hz"
+                }
+            },
+            1: {
+                name: "ONESHOT125",
+                defaultRate: 1000,
+                rates: {
+                    400: "400Hz",
+                    1000: "1kHz",
+                    2000: "2kHz",
+                    4000: "4kHz"
```

Comment:
4 kHz is impossible with OneShot125 - longest pulse of OS125 is 250us which doesn't leave any time for a inter-pulse pause. 3kHz should be possible, but I never tested.